### ISSUE TITLE
dw_client: ability to read scenario(s) from script(s)

### ISSUE THREAD

**Issue by tomcucinotta** [tomcucinotta/distwalk#32]
In order to specify a workload scenario in dw_client, in addition to providing command-line arguments, it might be useful to have the possibility to specify a "script" file, e.g.:
```
  dw_client -f script.dw
```

where script.dw could have the same exact syntax as the command-line arguments, perhaps with some more comfortable "dashless" way of specifying commands, e.g., the command
```
  -C 10000 -L 16384 -rs 512
```
might be specified with a script file like
```
#!dw_client -f

COMPUTE 10000us
LOAD 16KB
REPLY 512B
```

where, adding the "shell bang" first line, would allow for launching directly the script from the shell, making it executable.

The sharp symbol "#" could be used throughout the script to add comments that would be completely ignored by dw_client.

**Comment by tomcucinotta** [comment]
it was longer to explain than to do it (ok, the script syntax might be improved, but stil...)

**Comment by tomcucinotta** [comment]
Unfortunately, this is now broken with the latest commits.

**Comment by deRemo** [comment]
fixed in (squashed) commit fb210d54a0766d4d0339aa529ec7826d145ea16d
